### PR TITLE
BrickHack 8 - Schedule

### PR DESCRIPTION
**Pull request by sjmiller7**
For #1279 

Currently on draft for design

### CODE REVIEW

**Comment by peterkos** [comment]
The buttons jump around when clicking between the days -- could they be made to stay in the same position no matter the size of the schedule? maybe put the left one in its own flex column, or more likely set `flex-start` alignment? (or justify content, I always forget which is which)

https://user-images.githubusercontent.com/2782749/156395945-70561e36-3664-4ad5-b32f-4ac51813dfa9.mp4

File: index.js [hunk 1]
```diff
@@ -93,3 +93,147 @@ for (let i = 0; i < card.length; i++) {
...
+        output += ':' + String(date.getMinutes()).padStart(2, '0');
+    }
+    // AM/PM
+    if (date.getHours() >= 12) {
+        output += 'pm';
+    } else {
+        output += 'am';
+    }
+    return output;
+}
```
Comment (skyegallup) [inline index.js:239]: nit: missing newline at EOF
```suggestion
}

```

**Review by skyegallup — APPROVED** [review]
One nit, but otherwise looks okay - most of it is quite familiar from last year's code so I didn't look *too* too close

**Comment by skyegallup** [comment]
Also, just so we don't miss it: REMEMBER TO REMOVE SAMPLE EVENTS BEFORE PUSHING TO LIVE

**Review by skyegallup — APPROVED** [review]
👍 lgtm

**Pull request merged**